config/tls.go

```diff
@@ -22,7 +22,7 @@ func (t *TLS) MakeConfig(serverName string) (*tls.Config, error) {
 		return nil, nil
 	}
 
-	tlsConfig := &tls.Config{}
+	tlsConfig := &tls.Config{MinVersion: tls.VersionTLS12}
```

Comment:
Go redis does the same thing here too.
https://github.com/redis/go-redis/blob/master/options.go#L314-L317